src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -858,7 +858,7 @@ private static void CheckLambdaConversion(LambdaSymbol lambdaSymbol, TypeSymbol
                     lambdaSymbol.DeclaringCompilation,
                     member,
                     diagnostics,
-                    lambdaParameter.Locations.FirstOrDefault() ?? lambdaSymbol.SyntaxNode.Location);
+                    lambdaParameter.TryGetFirstLocation() ?? lambdaSymbol.SyntaxNode.Location);
```

Comment:
this commit should be the mechanical change of Locations.FirstOrDefault to TryGetFirstLocation()